include/passenger_details.h

```diff
@@ -1,35 +1,29 @@
-#include<stdio.h>
-#include<stdlib.h>
-#include<string.h>
-
 /**
  * @file passenger_details.h
  *
- * @brief Implementation of the linked list data structure.
- * Implements a linked-list for the purpose of storing the details of the passengers.
+ * @brief Reading the details of the passengers and storing the details into "passenger_details.csv" file
+ *
+ * @author Thejprabahth Rayasam- thejprabhathrayasam@cmail.carleton.ca
  */
 
-/**
- * @brief Defining the structure.
- */
+#ifndef passenger_details_h
+#define passenger_details_h
 
-struct details
-{
-    char name[20];    //Name of the passenger
-    int age;          //Age of the passenger
-    char gender[10];  //Gender of the passenger
-    char email[20];   //Email of the passenger
-    struct details *next;
-};
+/**
+* Declaring the name of the csv file
+*/
+#define filename "passenger_details.csv"
```

Comment:
Change the "filename" to some different name, since its conficting with the one in booking.c .

Reply:
Updated.. Thanks!!